src/app/views/main/main.component.html

```diff
@@ -1,22 +1,24 @@
 <div class="container">
 
   <div class="row">
-    <div class="paragraph col-md-12"></div>
+    <div class="testTextContainer col-md-12">
+      <span>{{ currentText }}</span>
+    </div>
   </div>
 
   <div class="row">
     <input type="text" placeholder="Type text here" class="form-control col-md-12" />
   </div>
 
   <div class="row justify-content-around">
-      <label for="selectDifficulty">Selected difficulty:</label>
-      <select class="form-control col-md-2" id="selectDifficulty">
+      <label>Selected difficulty:</label>
+      <select class="form-control col-md-2" #selectDifficulty>
         <option>Easy</option>
```

Comment:
Jeśli zmienisz nazwę pól to możesz zachować opcje w tym elemencie dodając parametr value: 

```
<select name="cars">
    <option value="volvo">Volvo XC90</option>
    <option value="saab">Saab 95</option>
    <option value="mercedes">Mercedes SLK</option>
    <option value="audi">Audi TT</option>
</select>